doc/api/console.markdown

```diff
@@ -85,15 +94,48 @@ Example:
     console.timeEnd('100-elements');
     // prints 100-elements: 262ms
 
-## console.trace(message[, ...])
+### console.trace(message[, ...])
 
 Print to stderr `'Trace :'`, followed by the formatted message and stack trace
 to the current position.
 
-## console.assert(value[, message][, ...])
+### console.assert(value[, message][, ...])
 
 Similar to [assert.ok()][], but the error message is formatted as
 `util.format(message...)`.
 
+## Class: Console
+
+<!--type=class-->
+
+Use `require('console').Console` or `console.Console` to access this class.
+
+    var Console = require('console').Console;
+    var Console = console.Console;
+
+You can use `Console` class to custom simple logger like `console`, but have
+different output streams.
+
+### new Console(stdout[, stderr])
+
+Create a new `Console` by passing one or two writable stream instances.
+`stdout` is a writable stream for print log or info output. `stderr`
+used for warning or error output. If `stderr` isn't passed, the warning
+and error output will be sent to the `stdout`.
+
+    var output = fs.createWriteStream('./stdout.log');
+    var errorOutput = fs.createWriteStream('./stderr.log');
+    // custom simple logger
+    var logger = new Console(output, errorOutput);
+    // use it like console
+    var count = 5;
+    logger.log('count: %d', count);
+    // in stdout.log: count 5
+
+The global `console` is a special `Console` that output is sent to
```

Comment:
'whose' seems ok : http://english.stackexchange.com/questions/23541/can-whose-refer-to-an-inanimate-object